src/main/java/build/buildfarm/BUILD

```diff
@@ -2,6 +2,8 @@ package(
     default_visibility = ["//src:__subpackages__"],
 )
 
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
```

Comment:
https://docs.bazel.build/versions/master/skylark/build-style.html#file-structure

I appreciate you moving it, but please refer to the above for ordering: load should precede `package`